ansible/roles/os_config/tasks/memory_optimization.yml

```diff
@@ -136,57 +136,28 @@
 
 - name: update nofile hard 1000
   pam_limits:
-    domain: '1000'
+    domain: '*'
```

Comment:
please split into 2 commuits .. 
1) handles the "user" issue i.e. 1000 -> "*" and b.t.w. this is not related to compose gravity .. it was just a "luck" that we had only single user that run the compose :) if we would create 2 users on server it wont work .. just first user is 1000

Reply:
@LazyDBA247-Anyvision  ping ?